src/wp-includes/rest-api/class-wp-rest-request.php

```diff
@@ -161,6 +161,18 @@ public function get_headers() {
 		return $this->headers;
 	}
 
+	/**
+	 * Determines if the request is the given method.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param string $method HTTP method.
+	 * @return bool Whether the request is of the given method.
+	 */
+	public function is_method( $method ) {
```

Comment:
```suggestion
	public function is_method( $method ): bool {
```

MInimum required PHP version allows the use of type hints, and for bool it is already in use at other places, so imho we should introduce this new function with a type hint.

Reply:
Thanks for the review.
There is currently no agreement on using return type hints in WordPress Core, so I’d refrain from using them for now. I’m fine with using them as soon as the Community reaches a consensus.

Reply:
The HTML API for example uses it, so there are already some examples for return types in core. But I get your point, so waiting for a consensus is fine with me.